_includes/manuals/1.10/4.3.5.3_gsstsig.md

```diff
@@ -32,8 +33,21 @@ Once the keytab file has been created, test it using kinit:
 If this works, clear the Kerberos ticket cache once again using `kdestroy`.
 
 Store the keytab at `/etc/foreman-proxy/dns.keytab`, ensure permissions are 0600 and the owner is `foreman-proxy`.
+If you are using SELinux, do not forget to update the file context.
 
-The DNS zone `Dynamic Updates` option on the DNS zones can now be set to `Secure Only`. Now follow the steps below under `Proxy Configuration`.
+Now update the configuration in /etc/foreman-proxy/settings.d/dns.yml for the smart proxy:
```

Comment:
This is already largely in the proxy configuration part, should we avoid adding this paragraph?  I don't know how clear it is that the proxy config section applies to both AD and FreeIPA.